TaskHouseUnitTests/FakeRepositories/FakeSkillRepository.cs

```diff
@@ -31,6 +31,11 @@ public FakeSkillRepository()
             }
             )
         { }
+
+        public IEnumerable<Skill> GetSkillsForWorker(int Id)
+        {
+            throw new NotImplementedException();
```

Comment:
Because it's not testable, or because you're being lazy? 0__0

Reply:
lazy

Reply:
Then I'm sorry hombre, write the test ❤️